Source/Particles/ElementaryProcess/QEDInternals/BreitWheelerEngineWrapper.H

```diff
@@ -130,10 +130,12 @@ public:
     AMREX_GPU_DEVICE
     AMREX_FORCE_INLINE
     int operator()(
-        const amrex::Real ux, const amrex::Real uy, const amrex::Real uz,
-        const amrex::Real ex, const amrex::Real ey, const amrex::Real ez,
-        const amrex::Real bx, const amrex::Real by, const amrex::Real bz,
-        const amrex::Real dt, amrex::Real& opt_depth) const noexcept
+        const amrex::ParticleReal ux, const amrex::ParticleReal uy,
+        const amrex::ParticleReal uz, const amrex::ParticleReal ex,
+        const amrex::ParticleReal ey, const amrex::ParticleReal ez,
+        const amrex::ParticleReal bx, const amrex::ParticleReal by,
+        const amrex::ParticleReal bz, const amrex::ParticleReal dt,
```

Comment:
```suggestion
        const amrex::ParticleReal bz, const amrex::Real dt,
```

Leave `dt` as `Real`.